packages/orchestration/test/exos/zcf-for-flows.test.ts

```diff
@@ -0,0 +1,185 @@
+import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+
+import { AmountMath, makeIssuerKit } from '@agoric/ertp';
+import { prepareSwingsetVowTools } from '@agoric/vow';
+import { makeFakeVatAdmin } from '@agoric/zoe/tools/fakeVatAdmin.js';
+import { makeZoeKitForTest } from '@agoric/zoe/tools/setup-zoe.js';
+import { makeNodeBundleCache } from '@endo/bundle-source/cache.js';
+import { E, Far } from '@endo/far';
+import type { TestFn } from 'ava';
+import { createRequire } from 'node:module';
+import { prepareZcfForFlows } from '../../src/exos/zcf-for-flow.js';
+import { provideDurableZone } from '../supports.js';
+
+const nodeRequire = createRequire(import.meta.url);
+const contractEntry = nodeRequire.resolve('../fixtures/zcfTester.contract.js');
+
+const makeTestContext = async () => {
+  let testJig;
+  const setJig = jig => (testJig = jig);
+  const fakeVatAdmin = makeFakeVatAdmin(setJig);
+  const { zoeService: zoe, feeMintAccess } = makeZoeKitForTest(
+    fakeVatAdmin.admin,
+  );
+
+  const bundleCache = await makeNodeBundleCache('bundles', {}, s => import(s));
+  const contractBundle = await bundleCache.load(contractEntry);
+
+  fakeVatAdmin.vatAdminState.installBundle('b1-contract', contractBundle);
+  const installation = await E(zoe).installBundleID('b1-contract');
+
+  const stuff = makeIssuerKit('Stuff');
+  await E(zoe).startInstance(installation, { Stuff: stuff.issuer });
+  assert(testJig, 'startInstance did not call back to setTestJig');
+
+  const zcf: ZCF = testJig.zcf;
+
+  const zone = provideDurableZone('root');
+  const vt = prepareSwingsetVowTools(zone);
+  const zcfLtd = prepareZcfForFlows(zcf, zone, vt);
+  return { zoe, zcf, stuff, feeMintAccess, zcfLtd, vt };
+};
+
+type TestContext = Awaited<ReturnType<typeof makeTestContext>>;
+
+const test = anyTest as TestFn<TestContext>;
+
+test.before('set up context', async t => (t.context = await makeTestContext()));
+
+test('yes as is: atomicRearrange(), ... getTerms()', async t => {
```

Comment:
took me a bit to understand the test name. consider,
- removed
- unchanged
- changed: <method-a>
- changed: <method-b>